contrib/charts/dragonfly/ci/command_extraargs-values.yaml

```diff
@@ -0,0 +1,6 @@
+command:
+  - /usr/local/bin/dragonfly
```

Comment:
what's the reason you separated stuff into lots of tiny files? :)

Reply:
For CI. I'm trying to prepare another PR right now, which will make use of [helm/chart-testing-action](https://github.com/helm/chart-testing-action). Helm's `ct` iterates over the `ci` subfolder of a chart and tries to provision the chart with each file. That's useful for testing different combinations :)